src/components/job-row.tsx

```diff
@@ -193,8 +175,13 @@ export function buildJobRow(
           })
         }
       />
-      <DeleteButton
+      <RefillButton
         job={job}
+        environmentList={environmentList}
+        showCreateJob={showCreateJob}
+      />
+      <DeleteWithConfirmationIcon
```

Comment:
```suggestion
      <ConfirmDeleteButton
```

Reply:
As described above, renamed `DeleteWithConfirmationButton` element into `ConfirmDeleteButton` and `DeleteWithConfirmationIcon` into `ConfirmDeleteIcon`